daemon/process/settings/dobby.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
@TeknoVenus  dobby.json is used only when using dobbyspec.json and not the OCI bundle, correct ?

Reply:
The `paths`, `logging` and `network` settings are used to configure the daemon, and apply for bundle and dobbyspec containers. Those settings should always be present

The remaining settings only apply to dobbyspec containers, so are not necessary on non-sky builds (hence why I removed them from the generic RDK config)